Tests/SwiftDocCTests/Infrastructure/AutomaticCurationTests.swift

```diff
@@ -16,13 +16,19 @@ import XCTest
 class AutomaticCurationTests: XCTestCase {
     func testAutomaticTopics() throws {
         // Create each kind of symbol and verify it gets its own topic group automatically
+        let decoder = JSONDecoder()
+        
         for kind in AutomaticCuration.groupKindOrder where kind != .module {
+            if !SymbolGraph.Symbol.KindIdentifier.allCases.contains(kind) {
```

Comment:
It's only applicable to this test.

Reply:
Have I understood correctly that this change is needed because new kinds (the extended types) were added to the auto curation list that aren't decodable in a main symbol graph file by default?

Reply:
If so, this test would be better if it decoded and transformed the extended symbols from an extension symbol graph file, the same way that they would be decoded and processed in a real documentation build.

I don't feel that this needs to be addressed in this PR but it would be good to file an issue and add a TODO/FIXME comment.